aws/resource_aws_budgets_budget_test.go

```diff
@@ -410,3 +522,68 @@ resource "aws_budgets_budget" "foo" {
 	t.Execute(&doc, budgetConfig)
 	return doc.String()
 }
+
+func testAccAWSBudgetsBudgetConfigWithNotification_Basic(budget budgets.Budget, notifications []budgets.Notification, emails []string, topics []string) string {
+	t := template.Must(template.New("t1").
```

Comment:
For consistency with the rest of the codebase for generating acceptance test configurations, this should be updated to use `fmt.Sprintf()` instead of `template`.